client/me/account/main.jsx

```diff
@@ -393,18 +406,18 @@ const Account = createReactClass( {
 	},
 
 	handleSubmitButtonClick() {
-		const { unsavedSettings } = this.props.userSettings;
+		const { unsavedUserSettings } = this.props;
 		this.recordClickEvent( 'Save Account Settings Button' );
-		if ( has( unsavedSettings, colorSchemeKey ) ) {
-			const colorScheme = get( unsavedSettings, colorSchemeKey );
+		if ( has( unsavedUserSettings, colorSchemeKey ) ) {
+			const colorScheme = get( unsavedUserSettings, colorSchemeKey );
 			this.props.recordTracksEvent( 'calypso_color_schemes_save', {
 				color_scheme: colorScheme,
 			} );
 			this.props.recordGoogleEvent( 'Me', 'Saved Color Scheme', 'scheme', colorScheme );
 			this.props.bumpStat( 'calypso_changed_color_scheme', colorScheme );
 		}
 
-		if ( has( unsavedSettings, 'language' ) ) {
+		if ( has( unsavedUserSettings, 'language' ) ) {
```

Comment:
Should we use `this.hasUnsavedUserSetting( 'language' )` here?

Reply:
Yes, we can do that 👍🏻

Reply:
See https://github.com/Automattic/wp-calypso/pull/49760/commits/1cc66281e1f35db279a3d90db5eb6e3ad70678bd